### PR TITLE
Set initial bounds based on locale

### PR DESCRIPTION
**Pull request by sorodrigo**
PT => Porto
ES => Madrid
SK => Slovakia
EN => Porto

### CODE REVIEW

File: app/controllers/map_controller.rb [hunk 1]
```diff
@@ -156,10 +156,28 @@ def set_labels
     }
   end
 
-  def pt_bounds
-    [
+  def default_bounds
+    pt_bounds = [
```
Comment (hugopeixoto) [inline app/controllers/map_controller.rb:160]: What do you think of doing something like:

```ruby
  BOUNDS = {
    "pt" => [
      [-8.693305501609501, 41.11531424472605],
      [-8.520026252042584, 41.17823491872437]
    ],
    "es" => [
      [-3.941566736714776,40.328785315750025],
      [-3.4482048268615415,40.515118259540344]
    ],
    "sk" => [
      [12.612613950622432,45.377461677566686],
      [16.88953848960196,46.848256073855225]
    ]
  }

  def default_bounds
    bounds.fetch(I18n.locale.to_s) { bounds["pt"] }
  end
```

Comment (sorodrigo) [reply app/controllers/map_controller.rb:160]: i think that's elegant asf 😄 , I'm just discovering ruby so pardon my french 😛 .

**Comment by sorodrigo** [comment]
> Shouldn't it be Slovakia?

I initially added `si` iso2 code because that's slovenia but then saw the url was using `sk`. I don't know which one it should be, the dropdown says:
Screen Shot 2020-04-25 at 00 50 49


But if it's slovakia then it should say: _Slovensko_ .

**Comment by sorodrigo** [comment]
My bad, I read `Slovensky` and sounded more `Slovenia`  to me than `Slovakia`. Will make the changes.

File: app/controllers/map_controller.rb (hunk 1 above)
Comment (simaob) [reply app/controllers/map_controller.rb]: Hugo is a great mentor =D

**Pull request merged**